app/assets/javascripts/angular/controllers/static_pages.js.erb

```diff
@@ -0,0 +1,13 @@
+app.controller('StaticPagesCtrl', ['$scope','$http', function($scope,$http){
+  $scope.message = "Hello World!";
+  $scope.services = "No Services";
+
+  // $http.get(url + '/services').success(function(response) {
+  //   if(services.nil?) {
```

Comment:
You can't do `services.nil?` in javascript! Also, try not to include commented out code in a PR